test/parallel/test-http-set-timeout-server.js

```diff
@@ -152,11 +152,7 @@ test(function serverResponseTimeoutWithPipeline(cb) {
 });
 
 test(function idleTimeout(cb) {
-  const server = http.createServer(common.mustCall((req, res) => {
-    req.on('timeout', common.mustNotCall());
-    res.on('timeout', common.mustNotCall());
-    res.end();
-  }));
+  const server = http.createServer();
```

Comment:
Could you add a comment about what this case it testing (that the timeout callback is called)?